client/src/App.tsx

```diff
@@ -28,8 +29,8 @@ function App(): JSX.Element {
           <RequestGroupDonorView requestGroupId="603d9b41eb57fc06447b8a23"/>
         </Route>
         <Route path='/'><DonorHomepage /></Route>
-      </Switch>
-    </Router>
+        </Switch>
+      </Router>
```

Comment:
nit: i don't think we should be making this spacing change, as the opening and closing switch and router tags no longer align